ethereum/truffle-config.js

```diff
@@ -165,6 +165,20 @@ module.exports = {
       gas: 4465030,
       gasPrice: 300000000000,
     },
+    karura: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.MNEMONIC,
+          // To use this local host, needed to run this: ENDPOINT_URL=wss://karura-rpc-1.aca-api.network npx @acala-network/eth-rpc-adapter@latest
+          "http://localhost:8545"
+          //"https://eth-rpc-karura.aca-api.network/"
```

Comment:
The previous comment is good, but it seems like you should remove this one. I'm generally never a fan of adding commented out code. Add comments or code, not commented code if it can be avoided.

Reply:
I'm either way, leaning towards in favor, of leaving the mainnet url here. That said, I see your point about commented out code.

Reply:
I would like to leave this here.